utils/browser.py

```diff
@@ -99,6 +100,8 @@ def start(webdriver_name=None, base_url=None, **kwargs):
         webdriver_name = browser_conf.get('webdriver', 'Firefox')
     webdriver_class = getattr(webdriver, webdriver_name)
 
+    webdriver_class.file_detector = UselessFileDetector()
```

Comment:
So, this cannot work because this assigns the detector to the class, but the webdriver sets its own upon instantiating, so it is necessary to move this to line 152 to assign this to the `browser` variable.

However, by doing this you will paralyze all tests that utilize file uploading, so this is a double edged sword :)